qlib/workflow/__init__.py

```diff
@@ -34,8 +37,13 @@ def start(self, experiment_name=None, recorder_name=None):
             name of the experiment one wants to start.
         recorder_name : str
             name of the recorder under the experiment one wants to start.
+        uri : str
+            The tracking uri of the experiment, where all the artifacts/metrics etc. will be stored.
+            The default uri are set in the qlib.config. Note that this uri argument will not change the one defined in the config file.
```

Comment:
We should have some discussions about this new argument

Reply:
Following the discussion, I have added a set_uri function to make uri consistent with client. And also add _default_uri and _current_uri to keep the uri in different status.